### PR TITLE
[1LP][RFR] Fixing ansible service order credentials test to find correct view to look up creds dropdown

### PR DESCRIPTION
**Pull request by kedark3**
Purpose or Intent
=================

__Fixing__ test_service_ansible_playbook_order_credentials as it was failing due to incorrect is_displayed function, and view couldn't find args. 

{{pytest: cfme/tests/ansible/test_embedded_ansible_services.py  --long-running -s -k 'test_service_ansible_playbook_order_credentials'}}

### CODE REVIEW

File: cfme/services/service_catalogs/ui.py [hunk 1]
```diff
@@ -149,8 +149,8 @@ class OrderServiceCatalogView(OrderForm):
     @property
     def is_displayed(self):
         return (
-            self.in_explorer and self.service_catalogs.is_opened and
```
Comment (kedark3) [inline cfme/services/service_catalogs/ui.py]: It was not able to find in_explorer hence changed it to in_service_catalogs.

File: cfme/services/service_catalogs/ui.py [hunk 2]
```diff
@@ -149,8 +149,8 @@ class OrderServiceCatalogView(OrderForm):
...
-            self.title.text == 'Order Service "{}"'.format(self.context['object'].name)
+            self.in_service_catalogs and self.service_catalogs.is_opened and
+            'Service "{}"'.format(self.context['object'].name) in self.title.text
```
Comment (kedark3) [inline cfme/services/service_catalogs/ui.py]: In latest builds word 'Order' has been removed from the title text hence changed it to this, it is backward compatible.

File: cfme/tests/ansible/test_embedded_ansible_services.py [hunk 3]
```diff
@@ -325,11 +325,11 @@ def test_service_ansible_playbook_order_credentials(ansible_catalog_item, ansibl
             "machine_credential": ansible_credential.name
         }
     view = navigate_to(service_catalog, "Order")
-    wait_for(lambda: view.machine_credential.is_displayed, timeout=5)
+    view.wait_displayed()
```
Comment (kedark3) [inline cfme/tests/ansible/test_embedded_ansible_services.py]: I was not entirely sure about .wait_displayed() and (view.fields('credential')).visible_widget and please let me know if there is a better way to accomplish this.

File: cfme/services/service_catalogs/ui.py [hunk 4]
```diff
@@ -63,7 +63,7 @@ class fields(ParametrizedView):  # noqa
         param_input = Input(id=ParametrizedString("param_{key}"))
         dropdown = VersionPick({
             Version.lowest(): BootstrapSelect(Parameter("key")),
-            "5.9": DialogFieldDropDownList(ParametrizedLocator(".//div[@input-id='{key|quote}']"))
+            "5.9": DialogFieldDropDownList(ParametrizedLocator(".//div[@input-id={key|quote}]"))
```
Comment (mfalesni) [inline cfme/services/service_catalogs/ui.py]: This should be on reviewers' checklists :)

**Pull request merged**